x-pack/plugins/security_solution/public/overview/components/entity_analytics/risk_score/index.tsx

```diff
@@ -273,12 +279,13 @@ const EntityAnalyticsRiskScoresComponent = ({ riskEntity }: { riskEntity: RiskSc
               <RiskScoreDonutChart severityCount={severityCount ?? EMPTY_SEVERITY_COUNT} />
             </EuiFlexItem>
             <EuiFlexItem>
-              <BasicTableWithoutBorderBottom
+              <StyledBasicTable
                 responsive={false}
                 items={data ?? []}
                 columns={columns}
                 loading={isTableLoading}
                 id={entity.tableQueryId}
+                rowProps={setTableRowProps}
```

Comment:
this can be inline, no reason to put in `useCallback`

```suggestion
                rowProps={() => ({
                  className: 'EntityAnalyticsTableHoverActions'
                })}
```

Reply:
actually doesn't even need to be a function
```suggestion
                rowProps={{
                  className: 'EntityAnalyticsTableHoverActions',
                }}
```

Reply:
Thanks for the suggestion. It looks much better.